README.md

```diff
@@ -43,26 +43,26 @@ Releases are available on the Jenkins update center.
 You need to be running a sufficiently recent Jenkins release: an LTS in the 1.580.x line or newer (currently 1.596.x for the latest updates), or a weekly release.
 See the [changelog](CHANGES.md) for news.
 
-For OSS Jenkins users, install _Workflow: Aggregator_ (its dependencies will be pulled in automatically).
+For OSS Jenkins users, install _Pipeline_ (its dependencies will be pulled in automatically).
 You will need to restart Jenkins to complete installation.
 
-CloudBees Jenkins Enterprise users get Workflow automatically as of the 14.11 (1.580.1.1) release.
-Otherwise install _CloudBees Workflow: Aggregator_ from the update center.
+CloudBees Jenkins Enterprise users get Pipeline automatically as of the 14.11 (1.580.1.1) release.
+Otherwise install _CloudBees Pipeline_ from the update center.
 Again dependencies will be pulled in automatically, including all the OSS plugins.
 
 # News & questions
 
 * [Changelog](CHANGES.md)
-* [jenkins-workflow tag](http://stackoverflow.com/tags/jenkins-workflow) on StackOverflow
+* [jenkins-pipeline tag](http://stackoverflow.com/tags/jenkins-pipeline) on StackOverflow
```

Comment:
:bug: do not change this now—there is no such tag yet. If and when that rename happens, I will update this line of course.